pkg/controller/volume/cache/desired_state_of_world.go

```diff
@@ -57,7 +57,7 @@ type DesiredStateOfWorld interface {
 	// should be attached to the specified node, the volume is implicitly added.
 	// If no node with the name nodeName exists in list of nodes managed by the
 	// attach/detach attached controller, an error is returned.
-	AddPod(podName types.UniquePodName, volumeSpec *volume.Spec, nodeName string) (api.UniqueVolumeName, error)
+	AddPod(pod *api.Pod, volumeSpec *volume.Spec, nodeName string) (api.UniqueVolumeName, error)
```

Comment:
I'd prefer to leave it as is. One of the reasons to have an alias is to hide the implementation, so the fact that it is a UID underneath or name should be irrelevant to than name since we retain the right to swap out the implementation as needed.